src/Microsoft.AspNet.Server.Kestrel/Networking/Libuv.cs

```diff
@@ -216,6 +227,40 @@ public void tcp_bind(UvTcpHandle handle, ref SockAddr addr, int flags)
         {
             handle.Validate();
             Check(_uv_tcp_bind(handle, ref addr, flags));
+            if (PlatformApis.IsWindows)
+            {
+                tcp_bind_windows_extras(handle);
```

Comment:
No is only off by default for "just in-case" compat; and doesn't seem to have side effect; both sockets need it activated for it to trigger (http://blogs.technet.com/b/wincat/archive/2012/12/05/fast-tcp-loopback-performance-and-low-latency-with-windows-server-2012-tcp-loopback-fast-path.aspx)

> To enable the fast path: 

> 1. Both ends of the TCP session must enable the capability by setting the SIO_LOOPBACK_FAST_PATH IOCTL.
2. The process initiating the TCP connection must set the SIO_LOOPBACK_FAST_PATH IOCTL **prior** to establishing the TCP session.
3. The target of the connection request must set the SIO_LOOPBACK_FAST_PATH IOCTL on the *listen socket*, that is, **prior** to accepting the connection.

And

> The default behavior of the TCP loopback interface is unchanged in Windows Server 2012 thereby preserving compatibility.   Adding support for the loopback fast path enhancement to an existing or new application requires adding the code to set the SIO_LOOPBACK_FAST_PATH  IOCTL in the manner described above.

Unless didn't do it on an specific ip bind; rather than all ip addresses, and localhost bind.